plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/file/TestIcebergFileMetastoreCreateTableFailure.java

```diff
@@ -43,8 +44,10 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+import static org.junit.jupiter.api.parallel.ExecutionMode.SAME_THREAD;
 
 @TestInstance(PER_CLASS)
+@Execution(SAME_THREAD) // testException is shared mutable state
```

Comment:
This test used to be marked `@Test(singleThreaded = true) // testException is a shared mutable state` and it got lost when migrating to JUnit. thanks @ebyhr for adding it back

are there any other cases where such annotation was lost?